.github/workflows/prettier-html.yml

```diff
@@ -0,0 +1,37 @@
+name: Pretty HTML
```

Comment:
I think it's best to use a name that is more meaningful. Idk something like `Prettify built site` or something like that that makes clear what exactly is being modified.

Reply:
Does ```HTML Formatter``` sound good?
or perhaps ```Structure HTML```?

Reply:
The point is which html? Because we already have a [prettier](https://github.com/alshedivat/al-folio/blob/master/.github/workflows/prettier.yml) GitHub action that tries to format the code. I think it should make clearer which HTMLs it is formatting, which are the ones that are the result of the page built. Maybe something like `Prettify gh-pages`?

Reply:
Oo let's go with that.